worlds/stardew_valley/__init__.py

```diff
@@ -270,7 +271,32 @@ def generate_basic(self):
         pass
 
     def get_filler_item_name(self) -> str:
-        return "Joja Cola"
+        if not self.filler_item_pool_names:
+            self.generate_filler_item_pool_names()
+        return self.multiworld.random.choice(self.filler_item_pool_names)
+
+    def generate_filler_item_pool_names(self):
+        include_traps, exclude_island = self.get_filler_item_rules()
+        available_filler = get_all_filler_items(include_traps, exclude_island)
+        available_filler = remove_limited_amount_packs(available_filler)
+        self.filler_item_pool_names = [item.name for item in available_filler]
+
+    def get_filler_item_rules(self):
+        if self.player in self.multiworld.groups:
+            link_group: ItemLinkGroup = self.multiworld.groups[self.player]
+            include_traps = True
+            exclude_island = False
+            for player in link_group["players"]:
+                player_options = self.multiworld.worlds[player].options
+                if not isinstance(player_options, StardewValleyOptions):
```

Comment:
Would it not be faster to just check the game of the player directly? Would be faster than a `isinstance` usage.

```suggestion
                if self.multiworld.game[player] != "Stardew Valley":

```

Reply:
Fair enough, I didn't think of it.